riot_api/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""
+Data processing the data from riot API
+"""
+
+
+import os
+from riotwatcher import LolWatcher
+
+from dotenv import load_dotenv
+
+
+from db.db import Session
+
+
+session = Session()
+
+load_dotenv()
+RIOTAPIKEY = os.getenv("RIOT_API_KEY")
+
+watcher = LolWatcher(RIOTAPIKEY)
+MY_REGION = "na1"
+
+# pylint: disable=wrong-import-position
+from .methods import *
```

Comment:
moved original `riot.py` into modules. 

We have sub directory `methods/` that will have all sub functions for API requests. 
This file just imports and exports everything